test/Feature/SingularSpec.hs

```diff
@@ -277,18 +301,24 @@ spec =
                 , matchHeaders = [matchContentTypeSingular]
                 }
 
-      it "executes the proc exactly once per request" $ do
```

Comment:
This test description is just non-sense here. I think it was copied from another test, but the test does something else. corrected that.